source/ranges.tex

```diff
@@ -12119,6 +12147,29 @@
 Equivalent to: \tcode{return -(y - x);}
 \end{itemdescr}
 
+\begin{itemdecl}
+friend constexpr range_rvalue_reference_t<V> iter_move(const @\exposid{inner-iterator}@& i)
+  noexcept(noexcept(ranges::iter_move(*i.@\exposid{parent_}@->@\exposid{current_}@)));
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Equivalent to: \tcode{return ranges::iter_move(*i.\exposid{parent_}->\exposid{current_});}
+\end{itemdescr}
+
+\begin{itemdecl}
+friend constexpr void iter_swap(const @\exposid{inner-iterator}@& x, const @\exposid{inner-iterator}@& y)
+  noexcept(noexcept(ranges::iter_swap(*x.@\exposid{parent_}@->@\exposid{current_}@, *y.@\exposid{parent_}@->@\exposid{current_}@)))
+  requires @\libconcept{indirectly_swappable}@<iterator_t<V>>;
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Equivalent to: \tcode{ranges::iter_swap(*x.\exposid{parent_}->\exposid{current_}, *y.\exposid{parent_}->\exposid{current_});}
```

Comment:
```suggestion
Equivalent to: \tcode{ranges::iter_swap(*x.\exposid{parent_}->\exposid{current_}, *y.\exposid{parent_}->\exposid{current_})}.
```

Reply:
Actually no; when we introduce code with "Equivalent to:" (note the trailing ":"), the code is supposed to be complete, as if in a codeblock that could be substituted. Your suggestion would apply if the code had been introduced using "Equivalent to", the idea being that later reads as a statement. That said, if this should be an expression and not a statement, then the ";" should be removed. Alternatively, we could use your suggestion and introduce it with "Equivalent to".  What do you prefer?

Reply:
Ah, yes. Again, I forgot to remove the colon, too. Edited.

Reply:
I'm fine leaving it as proposed. We could make the change, but we don't _have_ to, and that way we retain some symmetry between the two instances.